### PR TITLE
No animation -> no chat effect

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Technical
- [ ] Other :

## Content

If the user does not want to see animation, no chat effect will be performed. Also no account creation celebration will e displayed.

## Motivation and context

It will speed up the UI test, which have animation disabled.

It can also be useful to user with health condition (see #5557)

I tried to update the speed for Konfetti regarding the animation scale, but I think it has to be handled by the third party library. Same for the snowfall.

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/popup/PopupAlertManager.kt [hunk 1]
```diff
@@ -218,7 +218,7 @@ class PopupAlertManager @Inject constructor(
         if (!alert.isLight) {
             clearLightStatusBar()
         }
-        val noAnimation = !animate || isAnimationDisabled(activity)
+        val noAnimation = !(animate && activity.isAnimationEnabled())
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/popup/PopupAlertManager.kt:221]: I've read it several times, but please double check this change :)

Comment (fedrunov) [reply vector/src/main/java/im/vector/app/features/popup/PopupAlertManager.kt:221]: took me some time to proceed, but looks correct

**Review by fedrunov — APPROVED** [review]
LGTM

**Comment by bmarty** [comment]
(the label next release was added by mistake)

**Review by onurays — APPROVED** [review]
LGTM!

**Pull request merged**